### ISSUE TITLE
Unable to launch NVDA from runnvda.bat

### ISSUE THREAD

**Issue by noorkaif97** [nvaccess/nvda#15294]
Hi Team,

I have cloned the repo and then I tried to launch nvda from runnvda.bat command but nothing happened. Could you please help me resolve this issue. Attaching the command prompt screenshot for reference.

![image](https://github.com/nvaccess/nvda/assets/41202038/0a8500d7-7f55-4a91-8a04-2c7c9c0162f6)
